pom.xml

```diff
@@ -95,6 +95,7 @@
     <module>sql/catalyst</module>
     <module>sql/core</module>
     <module>sql/hive</module>
+    <module>sql/pipelines/core</module>
```

Comment:
QQ: will there be new modules under `sql/pipelines` other than core?

Reply:
No plans at the moment

Reply:
Then shall we simply make it `sql/pipelines`?